tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/Examples/InjectionTest/inject.lg

```diff
@@ -0,0 +1,11 @@
+﻿> !# @strict = false
+> !# @registerNamespace = foo
+> !# @registerAsFunctions = bar, cool
```

Comment:
>registerAsFunctions [](start = 6, length = 19)

I would name this "Exports"or "Functions"
```
!# @Exports = bar, cool
!# @Functions = bar, cool
```

Reply:
Now is Exports